src/modules/map/i18n/map.provider.js

```diff
@@ -9,7 +9,8 @@ export const provide = (lang) => {
 				map_info_button: 'Information',
 				map_info_button_help: 'Help',
 				map_info_button_contact: 'Contact',
-				map_info_button_about: 'About us'
+				map_info_button_about: 'About us',
+				map_zoom_extent_button: 'Show whole map'
```

Comment:
I would prefer: 'Zoom to full extent'

Reply:
Thanks, that's a good idea !

Reply:
@EnBakir Can you change that asap, please? We need the infrastructure code...😉